src/postprocessing.py

```diff
@@ -1,7 +1,15 @@
 import pandas as pd
-
 from steppy.base import BaseTransformer
 
+from src.logging import LOGGER
+from .pipeline_config import params
+from .utils import visualize_bboxes, get_class_mappings
+import os
+import PIL
+import numpy as np
+
+codes2names, names2codes = get_class_mappings(mappings_file=params.class_mappings_filepath)
```

Comment:
@i008 If we want those to be public shouldn't we make it explicit with `CODES2NAMES` and `NAMES2CODES` names?